com.unity.render-pipelines.high-definition/Runtime/Lighting/AtmosphericScattering/Fog.cs

```diff
@@ -51,28 +47,50 @@ public class Fog : VolumeComponent
         /// <summary>Enable volumetric fog.</summary>
         [DisplayInfo(name = "Volumetric Fog")]
         public BoolParameter enableVolumetricFog = new BoolParameter(false);
-        /// <summary>Volumetric fog anisotropy.</summary>
-        public ClampedFloatParameter anisotropy = new ClampedFloatParameter(0.0f, -1.0f, 1.0f);
+        // Common Fog Parameters (Exponential/Volumetric)
+        /// <summary>Fog albedo.</summary>
+        public ColorParameter albedo = new ColorParameter(Color.white);
         /// <summary>Multiplier for ambient probe contribution.</summary>
         [DisplayInfo(name = "Ambient Light Probe Dimmer")]
         public ClampedFloatParameter globalLightProbeDimmer = new ClampedFloatParameter(1.0f, 0.0f, 1.0f);
-
-        /// <summary>Sets the distance (in meters) from the Camera's Near Clipping Plane to the back of the Camera's volumetric lighting buffer.</summary>
-        [Tooltip("Sets the distance (in meters) from the Camera's Near Clipping Plane to the back of the Camera's volumetric lighting buffer.")]
+        /// <summary>Sets the distance (in meters) from the Camera's Near Clipping Plane to the back of the Camera's volumetric lighting buffer. The lower the distance is, the higher the fog quality is.</summary>
         public MinFloatParameter depthExtent = new MinFloatParameter(64.0f, 0.1f);
+        /// <summary> Controls which version of the effect should be used. </summary>
+        [Tooltip("Defines which technique is used for denoising the volumetric fog. Reprojection is very effective for static lighting, but can lead to severe ghosting for highly dynamic lighting. Gaussian is a good alternative for dynamic lighting. Using both techniques can give high quality results but will increase significantly the overall cost of the effect.")]
```

Comment:
The tooltip needs to be shorter, we can store this information in the manual/api docs. This can just be:
Controls which denoising technique to use for the volumetric effect.